masterlist.yaml

```
@@ -14838,14 +14846,15 @@ plugins:
         condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
       - name: 'WACCF_Armor and Clothing Extension.esp'
         condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
+      - name: 'CollegeOfWinterholdImmersive.esp'
+        condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
     inc:
       - 'accessible saarthal and labyrinthian.esp'
       - 'ApepiBetterCollege.esp'
       - 'Better College Students Room.esp'
       - 'College Guards SE JMR.esp'
       - 'CollegeDormDoors.esp'
       - 'CollegeGrounds.esp'
-      - 'CollegeOfWinterholdImmersive.esp'
       - 'CollegeofWinterholdUpgraded.esp'
```

We could still keep the inc I think, just turn it off if the patch is being used.
```suggestion
      - name: 'CollegeOfWinterholdImmersive.esp'
        condition: 'not active("UltimateCollege.esp")'
      - 'CollegeofWinterholdUpgraded.esp'
```